components/brave_wallet_ui/utils/amount.ts

```diff
@@ -241,7 +241,9 @@ export default class Amount {
 
     if (this.value === undefined || this.value.isNaN()) {
       return ''
-    } else if (this.value.decimalPlaces() < 2 || this.value.isGreaterThanOrEqualTo(10)) {
+    }
+    // @ts-ignore
```

Comment:
what are we ignoring here?

Reply:
Same as in other cases.

Reply:
Realised that we don't really need this `@ts-ignore`s - it was only my Webstorm IDE complaining. I've removed them.